test/suites/smoke/test-dynamic-fees.ts

```diff
@@ -304,7 +299,8 @@ describeSuite({
           log("Skipping test suite due to runtime version");
           return;
         }
-        const weightFee = ConstantStore(context).WEIGHT_FEE.get(specVersion.toNumber());
+        const runtime = paraApi.runtimeChain.toUpperCase();
```

Comment:
`paraApi.runtimeChain.toUpperCase()` will give the chain name like `MOONBASE ALPHA` or `MOONSAMA`.

We should instead use `api.consts.system.version.specName.toUpperCase()` which correctly gives the specname.

Reply:
Good catch, update it in a few places